GLTFSerialization/GLTFSerialization/GLTFHelpers.cs

```diff
@@ -320,6 +320,40 @@ public static void BuildMeshAttributes(ref Dictionary<string, AttributeAccessor>
 			}
 		}
 

```

Comment:
General question:
Do you need to do something in ConstructClip? I just pulled down your branch and loaded a couple sample models with morph targets and I did not see them animate. I had playAnimationOnLoad enabled.

Reply:
Specifically:
- https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/AnimatedMorphCube
- https://github.com/KhronosGroup/glTF-Sample-Models/tree/master/2.0/AnimatedMorphSphere

Reply:
This pull request only imports blend shapes, it does not import animation of blend shapes into Unity. For the examples posted above, they import into Unity with blend shapes - if you open the cube's SkinnedMeshRenderer component you can expand BlendShapes and manipulate the cube.
<img width="1401" alt="Screen Shot 2019-06-10 at 3 20 34 PM" src="https://user-images.githubusercontent.com/4493859/59230807-80457600-8b93-11e9-8fa6-3a5beafd0bc8.png">